### PR TITLE
[14.0][MIG] stock_picking_completion_info

### PR DESCRIPTION
**Pull request by hparfr**
Replace https://github.com/OCA/stock-logistics-warehouse/pull/1052

Depends on https://github.com/OCA/stock-logistics-warehouse/pull/1048

### CODE REVIEW

**Comment by hparfr** [comment]
@grindtildeath ready to review. 

I had to fix the tests with Form() for the immediate transfer and add an action_assign for the backorders.

**Comment by hparfr** [comment]
@simahawk can you please have a look on this one ?

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+stock-logistics-warehouse
```
Comment (simahawk) [inline oca_dependencies.txt]: this is not needed here as MQT=PIP

**Comment by hparfr** [comment]
Hi @simahawk can you update your review on this PR ?

**Comment by hparfr** [comment]
I did a rebase on v14 to trigger the CI and removed the test-dependencies.

**Review by grindtildeath — APPROVED** [review]
Thanks, LGTM :+1:

**Pull request merged**